app/views/bathrooms/show.html.erb

```diff
@@ -0,0 +1,25 @@
+<h1>Welcome to the Prime Porcelain page for <%= @bathroom.location %>'s Bathroom</h1>
+<h3>Average rating: <%= @bathroom.average_rating %></h3>
```

Comment:
Think about indentation - even in views. Good, consistent indentation can make code much easier to read.